howto/submit-assignments.md

```diff
@@ -0,0 +1,23 @@
+# 1 Million Women To Tech (1MWTT)
+
+## Submitting assignments for SOC2018 #HOWTO
+
+* Click on the respective week's assignment link and accept the assignment
+
+* Wait for a couple of minutes - a new branch called 'develop' should appear. Do your work in this branch
+
+* When you are ready to submit, create a pull request from the *develop* branch into the *master* branch.
+
+* A 'to-review' tag will be added to the repo, letting us know that your homework is ready to be corrected.
+
+* Once you have submitted, a mentor will review your homework by submitting a Pull Request Review;
+  * If you have met all the requirements of the homework, the mentor will 'approve' your homework - Congratulations!
+  * If you have not met all the requirements, the mentor will 'request changes' and give helpful comments on what you need to do to satisfy all the requirements. Do not worry! We learn most from our mistakes after all. <br> Once you have made the required changes create a new pull request like before (do not reopen the old one).
+  
+  > Note: Conventionally, pull requests get updated automatically if you make any commits to the compare branch (the branch you originally made changes to), so you don't have to create new pull requests if the original one is still open. However by creating a new pull request you let us know that your HW needs to be reviewed again.
+
+> Notes: 
+> - Sometimes there may be issues during branch creation, especially if there was an error or hold up during the creation of the repo by Github Classrooms. If the 'develop' branch does not appear within 15 minutes, please create and work on your own branch. **Please do not commit directly to master branch.**
+> - We are currently fixing an issue regarding our Github Bot and the Github API, once it is resolved you should be able to work in your own fork
```

Comment:
Actually I meant that they would be able to fork their new private repo from 1mwtt-soc1 to their own account so that they can work on as many branches as they like and send PRs between them without triggering HW submission via the bot